worker/src/routes/docs.ts

```diff
@@ -50,6 +54,22 @@ export function resolveURL(path: string, ctx: Context) {
 
   log.debug('[Docs/resolve] resolved path: ', resolvedPath);
 
+  if (branch) {
+    const req = await fetch(`https://api.github.com/repos/${PREVIEW_REPO_OWNER}/${PREVIEW_REPO_NAME}/commits/${ref}`, {
+      headers: {
+        // TODO replace user-agent and add Auth header to increase rate limits
+        'User-Agent': 'icaraps',
```

Comment:
@maxakuru see https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#user-agent-required why this is required

Reply:
if we need this with authenticated reqs then I vote to use the same sort of identifier as the Franklin urls: `prisma-cloud-docs--hlxsites` wdyt?